packages/runtime/src/controls/rich-text/rich-text.ts

```diff
@@ -74,31 +75,31 @@ export class RichTextControl extends PropController<RichTextControlMessage> {
       case RichTextControlMessageType.CHANGE_BUILDER_EDIT_MODE: {
         switch (message.editMode) {
           case BuilderEditMode.BUILD:
-            this.editor?.deselect().blur()
+            // this.editor?.deselect().blur()
             break
         }
         break
       }
       case RichTextControlMessageType.FOCUS: {
-        this.editor?.focus().moveToRangeOfDocument()
+        // this.editor?.focus().moveToRangeOfDocument()
```

Comment:
I wanted to get the editor working in the most basic sense that it functions as a `textArea`. these comments will be handled in future PRs